client/components/common/TestPlanResultsTable/index.jsx

```diff
@@ -140,7 +141,7 @@ const TestPlanResultsTable = ({
           {
```

Comment:
I'm noticing this TODO about using the assertion phrases from the db. @howard-e If this is still preferred, can you provide a little guidance on how to do that? I'm not currently seeing the new phrases when I inspect the graphQL calls in the browser, but maybe I'm overlooking something...